spec/observables/fromEvent-spec.ts

```diff
@@ -1,30 +1,34 @@
+/** @prettier */
 import { expect } from 'chai';
-import { expectObservable } from '../helpers/marble-testing';
 import { fromEvent, NEVER, timer } from 'rxjs';
 import { mapTo, take, concat } from 'rxjs/operators';
 import { TestScheduler } from 'rxjs/testing';
-
-declare const rxTestScheduler: TestScheduler;
+import { observableMatcher } from '../helpers/observableMatcher';
 
 /** @test {fromEvent} */
 describe('fromEvent', () => {
+  let rxTestScheduler: TestScheduler;
+
+  beforeEach(() => {
+    rxTestScheduler = new TestScheduler(observableMatcher);
+  });
+
   it('should create an observable of click on the element', () => {
-    const target = {
-      addEventListener: (eventType: any, listener: any) => {
-        timer(50, 20, rxTestScheduler)
-          .pipe(
-            mapTo('ev'),
-            take(2),
-            concat(NEVER)
-          )
-          .subscribe(listener);
-      },
-      removeEventListener: (): void => void 0,
-      dispatchEvent: (): void => void 0,
-    };
-    const e1 = fromEvent(target as any, 'click');
-    const expected = '-----x-x---';
-    expectObservable(e1).toBe(expected, {x: 'ev'});
+    rxTestScheduler.run(({ expectObservable, time }) => {
+      const delay1 = time('-----|     ');
+      const delay2 = time('     --|   ');
+      const expected = '   -----x-x---';
+
+      const target = {
+        addEventListener: (eventType: any, listener: any) => {
+          timer(delay1, delay2, rxTestScheduler).pipe(mapTo('ev'), take(2), concat(NEVER)).subscribe(listener);
```

Comment:
In run mode, you don't have to pass scheduler explicitly, thus `timer(delay1, delay2)` should work as well.

Reply:
Yes I missed that. Thanks for the hint. I removed the explicit scheduler argument.